openpype/lib/local_settings.py

```diff
@@ -35,6 +35,7 @@ def wrapper(*args, **kwargs):
 )
 
 from .import validate_mongo_connection
+from igniter.tools import (parse_get_url, parse_set_url)
```

Comment:
You can't import from igniter in `openpype.lib`. Igniter is available only in openpype process but not in any python host (Maya, Nuke, Houdini, etc.).

Reply:
Right now you have to move them to openpype and have copy of `parse_set_url` in igniter. We should create some folder in igniter that is added to pythonpath and sys.path where functions used for both build and code are.

NOTE: This PR will require new build.

Reply:
This is blocking issue for this PR to be merged. Currently is not possible to share code between igniter and OP.